markdown/counselors-submission-faq.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Session Submission FAQs
+description: Answers common questions around submitting a session to speak at THAT Conference.
+---
+
+We've collected a number of questions that we feel may (or have already been) frequently asked surrounding the process of submitting a session to speak at THAT Conference. If your question isn't answered here feel free to reach out in Slack in the #sessions channel or via e-mail at [Hello@THATConference.com](mailto:Hello@THATConference).
+
+### Why do I need a new account?
+
+After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including user accounts. You have to create a new account on the site.
+
+### What happened to my old sessions?
+
+All of the old session data has been deleted as the site is completely new this year. We may decide to migrate the old data over to the new site and connect it with your profile. Things are still under development so we're not 100% sure what will happen.
```

Comment:
Have this follow the same as account:
`After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including sessions.`
The continue with "We may decide to migrate..."